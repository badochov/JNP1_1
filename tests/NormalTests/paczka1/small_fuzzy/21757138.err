Error in line 4: KAZ S435 54,1
Error in line 8: ytWNtE S435 88,6
