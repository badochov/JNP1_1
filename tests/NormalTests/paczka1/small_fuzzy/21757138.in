?
ytWNtE S435 70,4
ytWNtE S435 55,4
KAZ S435 54,1
?S435
?
KAZ S34 8,6
ytWNtE S435 88,6
p8qzQR S34 12,0
M8y S886 30,4
ytWNtE S535 54,1
KAZ S34 16,7
KAZ S34 77,8
ytWNtE S535 131,5
M8y S886 113,1
ZqH S535 0,1