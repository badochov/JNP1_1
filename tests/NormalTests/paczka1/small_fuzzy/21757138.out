S435 15,0
ytWNtE S 15,0
S435 15,0
